examples/lapack_like/HermitianEig.cpp

```diff
@@ -7,12 +7,88 @@
    http://opensource.org/licenses/BSD-2-Clause
 */
 #include <El.hpp>
+
 using namespace std;
 using namespace El;
 
-// Typedef our real and complex types to 'Real' and 'C' for convenience
-typedef double Real;
-typedef Complex<Real> C;
+template<typename Real>
+void run_example(Int n, bool print)
```

Comment:
It isn't properly documented, but the `examples/` folder is meant to be more for demonstrating functionality, and the `tests/` folder is meant for correctness tests.

As an aside, Elemental uses `CamelCase` for function names rather than `snake_case`.

It might also be preferred to test *both* single-precision and double-precision in the same run (with the ideal case being able to individually disable each precision).